doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md

```diff
@@ -0,0 +1,50 @@
+#<a name="top"></a>Grouping Rules
+The file `grouping_rules.conf` can be instantiated from a template given in the Cygnus repository, `conf/grouping_rules.conf.template`.
```

Comment:
This line about how to confiure it can be put after saying "Such rules are jsut a Json within a configuration file (that can be instantiated from a template bla bla )"